test/mocha/unit/traffic-switch/traffic-switcher-i-token-fee-fetcher.test.ts

```diff
@@ -294,4 +309,50 @@ describe('TrafficSwitcherITokenFeeFetcher', () => {
       MetricLoggerUnit.Count
     )
   })
+
+  it('sampling traffic 100% should get results from both (Identical empty/0 fee), and return Current impl result', async () => {
```

Comment:
it seems like the `cleanResult` is well tested. Im not going to make it a blocking comment, if that;s the case.